tests/ibge/test_uf.py

```diff
@@ -16,3 +16,17 @@ def test_convert_code_to_uf(self):
         self.assertIsNone(convert_code_to_uf("00"))
         self.assertIsNone(convert_code_to_uf(""))
         self.assertIsNone(convert_code_to_uf("AB"))
+
+    def test_convert_uf_to_name(self):
+        # Testes para códigos válidos
+        self.assertEqual(convert_uf_to_name("SP"), "São Paulo")
+        self.assertEqual(convert_uf_to_name("RJ"), "Rio de Janeiro")
+        self.assertEqual(convert_uf_to_name("MG"), "Minas Gerais")
+        self.assertEqual(convert_uf_to_name("DF"), "Distrito Federal")
+        self.assertEqual(convert_uf_to_name("df"), "Distrito Federal")
+
+        # Testes para códigos inválidos
+        with self.assertRaises(ValueError):
+            convert_uf_to_name("XX")
+        with self.assertRaises(ValueError):
+            convert_uf_to_name("")
```

Comment:
Testando com tipo diferente de `str`

```suggestion
        with self.assertRaises(ValueError):
            convert_uf_to_name("")
        with self.assertRaises(ValueError):
            convert_uf_to_name(12)
```